graphql/tests/distros/data.json

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
Could you try using the data scrambler on this file by typing `make scramble file=graphql/tests/distros/data.json`? It should work now. You can undo it if doing so requires you to change too many other tests/files but it'd be nice to start using it

Reply:
![image](https://user-images.githubusercontent.com/4605522/90142825-da6b3980-dd4a-11ea-9e93-26442b234e7d.png)
I think i broke it